extension/config.release.json

```
@@ -0,0 +1,76 @@
+{
+  "defaultNetwork": "local",
+  "builtinNetworks": {
+    "mainnet": {
+      "blockExplorerUrl": "http://etherscan.io/",
+      "chainId": "0x1",
+      "displayName": "Mainnet",
+      "logo": "",
+      "rpcTarget": "https://mainnet.infura.io/v3/d0f317090d6645b6b494ddc6f1cce5ad",
+      "chainCurrencyName": "Ethereum",
+      "chainCurrency": "ETH",
+      "aggregatorUrl": "https://mainnet.blswallet.org",
+      "networkKey": "mainnet"
+    },
+    "local": {
+      "blockExplorerUrl": "N/A",
+      "chainId": "0x7a69",
+      "displayName": "Local Network",
+      "logo": "",
+      "rpcTarget": "http://localhost:8545",
+      "chainCurrencyName": "Ethereum",
+      "chainCurrency": "ETH",
+      "aggregatorUrl": "http://localhost:3000",
+      "networkKey": "local"
+    },
+    "arbitrum-testnet": {
+      "blockExplorerUrl": "https://rinkeby-explorer.arbitrum.io",
+      "chainId": "0x66eeb",
+      "displayName": "Arbitrum Test Network",
+      "logo": "",
+      "rpcTarget": "https://rinkeby.arbitrum.io/rpc",
+      "chainCurrencyName": "Arbitrum Ether",
+      "chainCurrency": "ARETH",
+      "aggregatorUrl": "https://arbitrum-testnet.blswallet.org",
+      "networkKey": "arbitrum-testnet"
+    },
+    "arbitrum": {
+      "blockExplorerUrl": "https://explorer.arbitrum.io",
+      "chainId": "0xa4b1",
+      "displayName": "Arbitrum One",
+      "logo": "",
+      "rpcTarget": "https://arb1.arbitrum.io/rpc",
+      "chainCurrencyName": "Ether",
+      "chainCurrency": "ETH",
+      "aggregatorUrl": "https://arbitrum.blswallet.org",
+      "networkKey": "arbitrum"
+    },
+    "optimism-kovan": {
+      "blockExplorerUrl": "https://kovan-optimistic.etherscan.io",
+      "chainId": "0x45",
+      "displayName": "Optimism Test Network",
+      "logo": "",
+      "rpcTarget": "https://kovan.optimism.io",
+      "chainCurrencyName": "Optimistic Kovan Ether",
+      "chainCurrency": "KOR",
+      "aggregatorUrl": "https://optimism-kovan.blswallet.org",
+      "networkKey": "optimism-kovan"
+    },
+    "optimism": {
+      "blockExplorerUrl": "https://optimistic.etherscan.io",
+      "chainId": "0xa",
+      "displayName": "Optimism",
+      "logo": "",
+      "rpcTarget": "https://mainnet.optimism.io",
+      "chainCurrencyName": "Ether",
+      "chainCurrency": "ETH",
+      "aggregatorUrl": "https://optimism.blswallet.org",
+      "networkKey": "optimism"
+    }
+  },
+  "currencyConversion": {
+    "api": "https://min-api.cryptocompare.com/data/price",
+    "apiKey": "injected-from-github-actions-environment",
```

```suggestion
    "apiKey": "CRYPTO_COMPARE_API_KEY",
```

Then we can inject with something like `envsubst` during release in GH actions. I will look into tweaking that park of the build process.